relay_test.go

```diff
@@ -490,6 +490,29 @@ func TestRelayConnection(t *testing.T) {
 	assert.Contains(t, err.Error(), errTest.Error(), "Unexpected error")
 }
 
+func TestSilentDrop(t *testing.T) {
```

Comment:
let's name this `TestRelayRateLimitDrop` (the tests are all in the same package, so the names should describe which component they're testing)